src/Components/Render.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Component.h"
+#include <mutex>
+#include <queue>
+#include <opencv2/opencv.hpp>
+
+class MyGLWidget;
```

Comment:
You might need to add a mock class for MyGLWidget in order to unit test this class.